### ISSUE TITLE
This Feels Unmaintained

### ISSUE THREAD

**Issue by Librechain** [mozilla/togetherjs#1181]
Which is really a shame because the idea is cool, but its so unmaintained that it feels outright **unsafe** for anyone to consider using / integrating into their actual workflows (any where). 

160 unresolved issues doesn't inspire any faith either. 

Guess this is one of those projects that will be shelved in perpetuity until someone decides to put forth a half-***** effort at some point in the future to bring the project back to its "glory days". 

Such is the world of developing, eh boys?

**Comment by gene1wood** [comment]
Indeed, in 2014 : https://www.ianbicking.org/blog/2014/09/professional-transitions.html
